test/unit/clients/adminusers_client/service/update_service_test.js

```diff
@@ -139,28 +133,21 @@ describe('adminusers client - patch request to update service', function () {
       'value': 'bar'
     }]
 
-    before(done => {
+    before(() => {
       provider.addInteraction(
```

Comment:
If you are removing `done` you should return the promise back up the chain
```suggestion
      return provider.addInteraction(
```